routes/github.js

```diff
@@ -20,6 +20,9 @@ exports.github = {
       //if push and deleted, do nothing
       return reply('skipped');
     }
+    if (payload.ref_type) {
```

Comment:
can we make it follow the same process as below - adding to dataToProcess and passing through that way.  There is a pretty big flaw in this in that if another hook comes in that doesn't have ref_type, then the existing env var is still set.  We should never explicitly set env vars while a process is running